autonomy/cli/helpers/chain.py

```diff
@@ -26,8 +26,10 @@
 from aea.configurations.data_types import PackageType
 from aea.configurations.loader import load_configuration_object
 from aea.crypto.base import Crypto, LedgerApi
-from aea.crypto.registries import make_crypto
+from aea.crypto.registries import crypto_registry, ledger_apis_registry
 from aea_ledger_ethereum.ethereum import EthereumApi, EthereumCrypto
+from aea_ledger_ethereum_hwi.exceptions import HWIError
+from aea_ledger_ethereum_hwi.hwi import EthereumHWIApi
```

Comment:
See below. I had requested that this not be a standard dependency but an `extra`